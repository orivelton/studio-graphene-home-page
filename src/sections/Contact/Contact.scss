@import '../../assets/scss/helper.scss';

.contact {
  background: $cinderella;
  display: flex;
  justify-content: center;
  flex-direction: row;
  padding: 82px 0;
  &__box {
    width: 536px;
  }

  &__description {
    max-width: 322px;
    margin-bottom: 10px;
    font-family: 'Gill Sans';
    color: $grey;
    font-size: 15px;
  }
}
