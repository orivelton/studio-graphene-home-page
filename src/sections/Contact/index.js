import React from 'react';
import './Contact.scss';
import Title from '../../components/Title';
import FormContact from '../../components/FormContact';

const Contact = () => (
  <section className="contact">
    <div className="contact__box">
      <Title textTop="COLLABORATE" text="WITH US" />
      <p className="contact__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
      <p className="contact__description">Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore</p>
    </div>
    <div className="contact__box">
      <FormContact />
    </div>
  </section>
);

export default Contact;
