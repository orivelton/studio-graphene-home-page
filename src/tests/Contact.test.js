import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '../sections/Contact';

test('renders correctly Contact', () => {
  const tree = renderer.create(<Contact />).toJSON();
  expect(tree).toMatchSnapshot();
});
