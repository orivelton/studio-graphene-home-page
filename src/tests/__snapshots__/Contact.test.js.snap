// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Contact 1`] = `
<section
  className="contact"
>
  <div
    className="contact__box"
  >
    <div
      className="title uppercase"
    >
      <h3
        className="title__top title__top--white"
      >
        COLLABORATE
        <span
          className="title__text title__text--line-white "
        >
          WITH US
        </span>
      </h3>
    </div>
    <p
      className="contact__description"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
    <p
      className="contact__description"
    >
      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
    </p>
  </div>
  <div
    className="contact__box"
  >
    <form
      className="form"
    >
      <span
        className="icon icondetail02_2x form__icon"
      />
      <div
        className="form__box"
      >
        <label
          className="form__label"
          htmlFor="name"
        >
          Name
        </label>
        <input
          className="form__input"
          id="name"
          placeholder="Name"
          type="text"
        />
        <label
          className="form__label"
          htmlFor="phone"
        >
          Phone
        </label>
        <input
          className="form__input"
          id="phone"
          placeholder="phone"
          type="tel"
        />
        <label
          className="form__label"
          htmlFor="experience"
        >
          Experience
        </label>
        <select
          className="form__select"
          id="experience"
        >
          <option>
            Select
          </option>
        </select>
      </div>
      <div
        className="form__box"
      >
        <label
          className="form__label"
          htmlFor="surname"
        >
          Surname
        </label>
        <input
          className="form__input"
          id="surname"
          placeholder="Surname"
          type="text"
        />
        <label
          className="form__label"
          htmlFor="email"
        >
          EMAIL ADDRESS
        </label>
        <input
          className="form__input"
          id="email"
          placeholder="email"
          type="email"
        />
        <label
          className="form__label"
          htmlFor="hear"
        >
          Wherw did you hear about us?
        </label>
        <select
          className="form__select"
          id="hear"
        >
          <option>
            Select
          </option>
        </select>
        <button
          className="form__btn uppercase"
        >
          Send
        </button>
      </div>
    </form>
  </div>
</section>
`;
